api/prisma/migrations/9_enable_all_parter_demographics/migration.sql

```diff
@@ -0,0 +1,3 @@
+
```

Comment:
Since this is the functionality in Doorway going forward, could you update the seed files to reflect this on yarn setup